Assets/Game/Scripts/CharacterInfo.cs

```diff
@@ -13,17 +13,18 @@ public class CharacterInfo : MonoBehaviour {
 	// More information on character stats https://github.com/warpfox/EB-Clone/wiki/Character-Stats
 	[SerializeField] private int iq, guts, luck, vitality, speed, offense, defense;
 	[SerializeField] private int characterLevel, currentEXP, maxHP, maxPP, currentHP, currentPP;
-  private int[] nessGrowthRates = new int[]{18,5,4,7,6,5,5};
```

Comment:
Wondering if we should have CharacterInfo as an abstract class and each character have their own implementation. That way, you just have a growthRates field and each concrete implementation of the class can assign their own unique values to it.